.github/workflows/build-test-push-workflow.yml

```diff
@@ -83,12 +83,11 @@ jobs:
       uses: aws-actions/amazon-ecr-login@v1
     - name: Make Splunk Operator Image
       run: |
-        docker pull registry.access.redhat.com/ubi8/ubi-minimal:latest
-        docker build -t ${{ secrets.ECR_REPOSITORY }}/${{ env.SPLUNK_OPERATOR_IMAGE_NAME }}:$GITHUB_SHA .
+        make docker-build IMG=${{ secrets.ECR_REPOSITORY }}/${{ env.SPLUNK_OPERATOR_IMAGE_NAME }}:$GITHUB_SHA
```

Comment:
Just a thought, Is there a way, we can use the `Install Operator SDK` from the make file? I think that is where some arch-specific values are there, and using it directly from the make file will be a lot of value in terms of maintaining all the supported platforms.

Reply:
we have plans to add that subba. I am still working on it. since testing on pipeline taking sometime It takes time.

Reply:
One other thing is, with this change, seems like we are duplicating the UT, as there is a separate UT check item. Probably,  the `test` phony can be decoupled from this path?

Reply:
this was always there , i changed the way to use make command and COMMIT id (which is in GITHUB_SHA)